cmd/romana.k8s

```diff
@@ -112,215 +112,164 @@ show_hosts() {
-		echo "Unable to fetch owners using URL '${tenant_url}${tenant_path}'"
+		echo "Unable to fetch tenants using URL '${tenant_url}${tenant_path}'"
 		return
 	fi
-	read -r owner_id _ < <(jq -r --arg name "$owner_name" '.[] | select(.Name==$name) | "\(.Id)"' <<< "$response")
+	read -r tenant_id _ < <(jq -r --arg name "$tenant_name" '.[] | select(.Name==$name) | "\(.Id)"' <<< "$response")
 
-	if ! [[ "$owner_id" ]]; then
-		echo "Unable to find owner id using owner ref '$owner_name'"
+	if ! [[ "$tenant_id" ]]; then
+		echo "Unable to find tenant id using tenant ref '$tenant_name'"
 		return
 	fi
 
-	post_data=$(printf '{"id": 0, "name": "%s"}' "$tier_name")
-	tier_path="$tenant_path/$owner_id/segments"
-	echo "Sending request to '${tenant_url}${tier_path}': '$post_data'"
-	curl -s -X POST -H "Content-Type: application/json" --data "$post_data" "${tenant_url}${tier_path}"; printf "\n"
+	post_data=$(printf '{"id": 0, "name": "%s"}' "$segment_name")
+	segment_path="$tenant_path/$tenant_id/segments"
+	echo "Sending request to '${tenant_url}${segment_path}': '$post_data'"
+	curl -s -X POST -H "Content-Type: application/json" --data "$post_data" "${tenant_url}${segment_path}"; printf "\n"
 }
 
 add_policy() {
 	if (( $# == 0 )) || [[ "$1" == "help" ]]; then
-		echo "romana add-policy <policy-file>"
+		echo "romana add-policy <tenant> <policy-file>"
 		return
 	fi
-	if (( $# != 1 )); then
-		echo "ERROR: expected 1 parameters, received $#"
+	if (( $# != 2 )); then
+		echo "ERROR: expected 2 parameters, received $#"
 		return
 	fi
-	local policy_file="$1"
+	local tenant="$1" policy_file="$2"
 	if ! [[ -f "$policy_file" ]]; then
 		echo "Unable to find file '$policy_file'"
 		return
 	fi
-	echo "Sending POST request to '${kube_apiserver_url}${romana_policy_path}': '@$policy_file'"
-	curl -s -X POST -H "Content-Type: application/json" --data "@$policy_file" "${kube_apiserver_url}${romana_policy_path}"; printf "\n"
+	local url=$(printf "${kube_apiserver_url}${romana_policy_path}" "$romana_policy_version" "$tenant")
+	echo "Sending POST request to '$url': '@$policy_file'"
+	curl -s -X POST -H "Content-Type: application/json" --data "@$policy_file" "$url"; printf "\n"
 }
 
 remove_policy() {
 	if (( $# == 0 )) || [[ "$1" == "help" ]]; then
-		echo "romana add-policy <policy-file>"
+		echo "romana remove-policy <tenant> <policy-name>"
 		return
 	fi
-	if (( $# != 1 )); then
+	if (( $# != 2 )); then
 		echo "ERROR: expected 1 parameters, received $#"
 		return
 	fi
-	local policy_name="$1"
-	echo "Sending DELETE request to '${kube_apiserver_url}${romana_policy_path}/$policy_name'"
-	curl -s -X DELETE "${kube_apiserver_url}${romana_policy_path}/$policy_name"; printf "\n"
+	local tenant="$1" policy_name="$2"
+	local url=$(printf "${kube_apiserver_url}${romana_policy_path}/$policy_name" "$romana_policy_version" "$tenant")
+	echo "Sending DELETE request to '$url'"
+	curl -s -X DELETE "$url"; printf "\n"
 }
 
-show_policy() {
```

Comment:
With the addition of namespaces to policies, it'd need to interrogate namespaces before listing the policies within them. And then it should logically inspect the versions ( [this bit](https://github.com/romana/romana/blob/feature/kube-namespaces/romana-install/roles/stack/kubernetes/postinstall/files/romana-thirdparty-resource.yaml#L6-L7) ) also.

Without a good way to handle those parts of it, I skipped it for now and let it handle just the add/remove policy that was useful for the demo script.
Eventually it'll be replaced by either the romana tool, kubectl functionality or both.